scripts/build.js

```diff
@@ -10,7 +10,8 @@ const {
   mkdirp,
   remove,
   writeFile,
-  writeJSON
+  writeJSON,
+  rename
```

Comment:
This doesn't seem to be being used?

Reply:
Ahh, right, tried it to use before and forgot to remove it 👍